.github/workflows/stale_prs_and_issues.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Close Stale PRs and Issues'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions: read-all
+
+jobs:
+  stale:
+    permissions:
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e # v9.0.0
+        with:
+          stale-issue-message: 'This issue will be closed in a week because of 9 months of no activity.'
+          stale-pr-message: 'This PR will be closed in a week because of 2 weeks of no activity.'
+          close-issue-message: 'This issue was closed because it has been stalled for 9 months with no activity.'
+          close-pr-message: 'This PR was closed because it has been stalled for 2 week with no activity.'
+          days-before-pr-stale: 14
+          days-before-issue-stale: 274
+          days-before-close: 7
+          ascending: true
+          exempt-pr-labels: 'no_stale'
```

Comment:
does it means that we should add this label for repo?

Reply:
Yes, I have created "no_stale" label for skipping